app/styles/components/post/post.note.css

```
@@ -21,9 +21,9 @@
     color: inherit;
     fill: currentcolor;
 
-    border: var(--border-width-thin) solid;
+    border: var(--cc-border-width-thin) solid;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)